website/docs/r/compute_network.html.markdown

```diff
@@ -43,8 +53,21 @@ The following arguments are supported:
     `auto_create_subnetworks` attribute. This attribute may not be used if
     `auto_create_subnetworks` is specified. This attribute is deprecated.
 
+* `peering` - (Optional) Adds a peering to the specified network. This can be specified
+    multiple times for multiple network peerings. Structure is documented below.
+
 * `project` - (Optional) The project in which the resource belongs. If it
     is not provided, the provider project is used.
+- - -
+
+The `peering` block supports:
+
+* `name` - (Required) Name of the peering.
+* `network` - (Required) Resource link of the peer network.
+* `auto_create_routes` - (Optional) If set to true, the routes between the two networks will
+  be created and managed automatically.
+* `state` - (Computed) State for the peering.
```

Comment:
Output-only attributes typically live under Attributes Reference, but I'm not sure exactly what it should look like because these are part of the `peering` block.